apps/condo/domains/user/components/UserMenu.tsx

```diff
@@ -41,14 +42,41 @@ export const UserMenu: React.FC<UserMenuProps> = ({
         router.push('/user')
     }, [router])
 
+    const authChannel = useMemo(() => {
+        if (typeof window === 'undefined' || typeof window.BroadcastChannel === 'undefined') return null
+        return new BroadcastChannel('auth')
+    }, [])
```

Comment:
Use `useBroadcastChannel` hook.
And check `typeof window.BroadcastChannel === 'undefined'` in it